src/components/Login/index.jsx

```diff
@@ -25,15 +22,14 @@ class Login extends React.Component {
 
   async componentDidMount() {
     const { token, redirect } = localStorage
-    const { queryString, history } = this.props
-    const queryParams = new URLSearchParams(queryString);
+    const queryParams = new URLSearchParams(this.props.location.search);
     const code = queryParams.has('code') ? queryParams.get('code') : '';
```

Comment:
this can be simplified. if the key is not in the URLSP object then it returns null

change to `const code = queryParams.get('code');`